react-native/ReactNativeFlipperExample/ios/ReactNativeFlipperExample/AppDelegate.m

```diff
@@ -12,35 +12,17 @@
 #import <React/RCTRootView.h>
 
 #ifdef FB_SONARKIT_ENABLED
-#import <FlipperKit/FlipperClient.h>
-#import <FlipperKitLayoutPlugin/FlipperKitLayoutPlugin.h>
-#import <FlipperKitNetworkPlugin/FlipperKitNetworkPlugin.h>
-#import <FlipperKitReactPlugin/FlipperKitReactPlugin.h>
-#import <FlipperKitUserDefaultsPlugin/FKUserDefaultsPlugin.h>
-#import <SKIOSNetworkPlugin/SKIOSNetworkAdapter.h>
-
-static void InitializeFlipper(UIApplication* application) {
-  FlipperClient* client = [FlipperClient sharedClient];
-  SKDescriptorMapper* layoutDescriptorMapper =
-      [[SKDescriptorMapper alloc] initWithDefaults];
-  [client addPlugin:[[FlipperKitLayoutPlugin alloc]
-                            initWithRootNode:application
-                        withDescriptorMapper:layoutDescriptorMapper]];
-  [client addPlugin:[[FKUserDefaultsPlugin alloc] initWithSuiteName:nil]];
-  [client addPlugin:[FlipperKitReactPlugin new]];
-  [client addPlugin:[[FlipperKitNetworkPlugin alloc]
-                        initWithNetworkAdapter:[SKIOSNetworkAdapter new]]];
-  [client start];
-}
+#import "ReactNativeFlipper.h"
 #endif
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication*)application
     didFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
-#ifdef FB_SONARKIT_ENABLED
-  InitializeFlipper(application);
-#endif
+  
+  #ifdef FB_SONARKIT_ENABLED
```

Comment:
You should ideally add one more check  of `DEBUG` as your class and method definition is  behind DEBUG flag. 

If we go back to the prev implementation of calling c function, then that would be better.